src/clients/__tests__/oidc-react.test.ts

```diff
@@ -167,4 +174,26 @@ describe('Oidc client ', () => {
       }
     });
   });
+  describe('setting autoSignIn=false ', () => {
+    beforeEach(() => {
+      setClientConfig({ ...config.client, autoSignIn: false });
+      initTests();
+    });
+    afterEach(() => {
+      clearTests();
+    });
+
+    it('changes init() and it does not call signinSilent, just getUser  ', async () => {
+      const email = 'autoSignIn@test.com';
+      mockMutator.setLoadProfilePayload(
+        mockMutator.createValidUserData({ email }),
+        undefined
+      );
+      await to(client.init());
+      // getInitCallCount() actually tracks if signInSilent is called, not client.init()
```

Comment:
If "getInitCallCount() actually tracks if signInSilent is called, not client.init()" as said, instead of commenting about surprising behavior, how about renaming stuff or doing something else in order to remove ambiguity?